.github/workflows/main.yml

```diff
@@ -0,0 +1,52 @@
+# Simple workflow for deploying static content to GitHub Pages
```

Comment:
Would be nice to remove default comments from the action if it was just part of the original source. We can keep comments that are adding value.

Reply:
Removed unnecessary comments